html/browsers/the-window-object/named-access-on-the-window-object/window-named-properties.html

```diff
@@ -25,6 +26,13 @@
   assert_equals(window["bar"],
                 document.getElementsByTagName("iframe")[0].contentWindow);
 }, "Static name");
+
+test(function() {
+  assert_true("quux" in window, "quux not in window");
+  assert_equals(window["quux"],
+                document.getElementsByTagName("iframe")[4]);
```

Comment:
Doesn't the spec require to return the nested browsing context's WindowProxy object, if there is one?

Reply:
There is one, but it doesn't have a name per <https://html.spec.whatwg.org/multipage/browsers.html#creating-a-new-nested-browsing-context>. (Note that the iframe has an `id` attribute.)